yb-voyager/src/namereg/namereg.go

```diff
@@ -109,28 +130,28 @@ func (reg *NameRegistry) Init() error {
 // TODO: only registry tables from tablelist, not all from schema.
 func (reg *NameRegistry) registerNames() (bool, error) {
 	switch true {
-	case reg.role == SOURCE_DB_EXPORTER_ROLE && reg.SourceDBTableNames == nil:
+	case reg.params.Role == SOURCE_DB_EXPORTER_ROLE && reg.SourceDBTableNames == nil:
```

Comment:
How about embedding the `*NameRegistryParams` in the `NameRegistry`? Then we won't need to qualify `reg.params` for every param access.

Reply:
This is a bit tricky. If i embed the fields, then fields like role get persisted automatically, which we don't want (because that will be different across processes). 
 Will look into this.